docs/workshop-hello-world.md

```diff
@@ -0,0 +1,127 @@
+# Purpose
+This is meant to be a small workshop that explores deploying a hello world application
+to the AWS EKS Kubernetes cluster, as well as exploring a few items available within
+the cluster.
+
+## Pre-Req
+- Install a tool to access the cluster in a user friendly way such as: https://k9scli.io/
+- Set up a SSO login session to access the cluster and login, for example: https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2632286259/AWS+SSM+Session+Manager#%5CuD83D%5CuDCD8-Log-in-to-an-AWS-account
+  - `aws sso login --profile dnt-dev-admin`
```

Comment:
This is implying I need access to the `org-sagebase-dnt-dev` account. Does this mean I can't access the cluster unless I have access to the Admin role on this account? I think it would be helpful to elaborate more on what this `dnt-dev-admin` profile is.

Reply:
I added notes here: https://github.com/Sage-Bionetworks-Workflows/eks-stack/pull/19/commits/5c6a70ff012b96f121863ccf5cba2c68a51b7786

For this workshop we are using the `org-sagebase-dnt-dev`, but any account with the EKS cluster deployed will work assuming you are using the right values.

> I think it would be helpful to elaborate more on what this dnt-dev-admin profile is.

This is what this: https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2632286259/AWS+SSM+Session+Manager#%5CuD83D%5CuDCD8-Log-in-to-an-AWS-account documentation is covering around SSO login sessions. I won't be copying documentation to this page since it's a big process itself and is already well documented in confluence.

Reply:
What you've added looks good, thanks!